docs/sources/troubleshoot/support_bundle.md

```diff
@@ -38,7 +38,8 @@ A support bundle contains the following data:
 `/api/v0/web/components` endpoint.
 * `alloy-logs.txt` contains the logs during the bundle generation.
 * `alloy-metadata.yaml` contains the {{< param "PRODUCT_NAME" >}} build version and the installation's operating system, architecture, and uptime.
-* `alloy-metrics.txt` contains a snapshot of the internal metrics for {{< param "PRODUCT_NAME" >}}.
+* `alloy-metrics-sample-1.txt` contains a snapshot of the internal metrics for {{< param "PRODUCT_NAME" >}} at the start of the bundle collection.
+* `alloy-metrics-sample-2.txt` contains a snapshot of the internal metrics for {{< param "PRODUCT_NAME" >}} at the end of the bundle collection.
```

Comment:
Yeah, I went back and forth on options, ended up going with 1/2 with the idea that future changes might support the ability to end up with additional samples, but it would probably be best to have them all have distinct uniquely identifiable names anyway. Will update.